src/resmom/start_exec.c

```diff
@@ -2315,6 +2315,8 @@ get_failed_moms_and_vnodes(job *pjob, int pipefd, int prolo_pipefd, vnl_t **vnl_
 			/* now get the actual string data */
 			r_buf = read_pipe_data(pipefd, r_size, 0);
 			if (r_buf != NULL) {
+				snprintf(log_buffer, sizeof(log_buffer), "got failed node host %s", r_buf);
```

Comment:
I'm not quite sure what this log message is trying to express.  Maybe rewording it?

Reply:
It's just a DEBUG3 message showing what the child mom (executing on behalf of the job) has received from the parent mom reporting what is sees as a failed node host. The node host was considered as failed as a prologue hook executing on the node has explicitly rejected the request, or the host has not received any acknowledgement that the prologue hook is ok or not ok within the 'job_launch_delay' amount of job startup. Maybe it's better if I change "got failed node host" to "received from parent mom that node host <node_name> is not healthy"